cheatsheets/Abuse_Case_Cheat_Sheet.md

```diff
@@ -101,7 +101,7 @@ Make a workshop that includes people with the following profiles:
 
 - **Business analyst**: Will be the business key people that will describe each feature from a business point of view.
 - **Risk analyst**: Will be the company's risk personnel that will evaluate the business risk from a proposed attack (sometimes it is the **Business analyst** depending on the company).
-- **Offsensives (Pentester or Application Security people with offensive mindset)**: Will be the *attacker* that will propose all attacks that he can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist (Pentester or AppSec consultant from a security firm). If possible, include 2 offensives people (ex: 1 Pentester + 1 AppSec) in order to increase the number of possible attacks that will be identified and considered.
+- **Offensives (Pentester or Application Security people with offensive mindset)**: Will be the *attacker* that will propose all attacks that he can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist (Pentester or AppSec consultant from a security firm). If possible, include 2 offensives people (ex: 1 Pentester + 1 AppSec) in order to increase the number of possible attacks that will be identified and considered.
```

Comment:
What do you think about this?
```suggestion
- **Security officer**: Will be the *attacker* that will propose attacks that they can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist. If possible, include 2 security officers with different backgrounds in order to increase the number of possible attacks that will be identified and considered.
```

Reply:
I think it's an improvement.
Officer? Engineer?

Reply:
If you're talking about an attacker, then "penetration tester" or "red team" would make more sense to me.

"security officer" is a bit vague - I don't really know what to expect from someone in that role, but I would think it's more on the defensive and governance side.

Reply:
Good point - the role here is really limited to Attacker. Whether an organization's "Security Officer" fulfills this role as part of a broader mandate is outside the scope of this recommended approach, where targeted/specific roles are required to accomplish the activities.

This role must be capable of brainstorming potential feasible attacks on the application.

Reply:
Please go forward with using "Penetration Tester"

Reply:
Sorry to be a bother; I submitted this change directly via the github web ui - I'm unfamilar with how to revise the merge request. Could you let me know how best to revise my change? Should I cancel and resubmit or is there a way to add a subsequent revision directly on the site without checking out the repo?

Reply:
You can go to the „Files changed” in this PR, click on three dots in the upper right corner and choose edit file.

Reply:
Or this link should also work: https://github.com/cmcconomy/CheatSheetSeries/edit/patch-1/cheatsheets/Abuse_Case_Cheat_Sheet.md?pr=%2FOWASP%2FCheatSheetSeries%2Fpull%2F437

Reply:
Thank you; second set of changes submitted.